### DISCUSSION TITLE
Feature request: ES export

### DISCUSSION THREAD

**Discussion by andreaslarssen** [FortAwesome/Font-Awesome#20904] [Feature Requests]
### Is your feature request related to a problem?

Using the `@fortawesome/free-brands-svg-icons` pacakge in an Angular app:

```
FILE_NAME depends on '@fortawesome/free-brands-svg-icons/ICON_NAME'. CommonJS or AMD dependencies can cause optimization bailouts.
```

### Feature description

The package not to use CommonJS exports

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by fish3046** [comment]
[FA claims to be fully built using ES modules](https://fontawesome.com/docs/apis/javascript/troubleshoot#tree-shaking-isn-t-working-or-performing-slowly).  I get this same build warning when using Angular as well.

From my digging, it seems the Angular build process uses static analysis to look for the `require()` statement.  If found it throws this warning.  [Lot's of icons have this require() call](https://github.com/FortAwesome/Font-Awesome/blob/6.x/js-packages/%40fortawesome/free-solid-svg-icons/faAdjust.js#L3).  [But not all of them](https://github.com/FortAwesome/Font-Awesome/blob/6.x/js-packages/%40fortawesome/free-solid-svg-icons/faChevronDown.js).

Oddly enough, Angular points to some icons in my project that _don't_ use `require()`.  I don't know enough about the internals to have a solution though.

**Comment by fish3046** [comment]
As a quick update, I've had some good luck with the shorted style:
```ts
import {faChevronUp} from '@fortawesome/free-solid-svg-icons';
```

vs the longer (supposedly the same)

```ts
import {faChevronUp} from '@fortawesome/free-solid-svg-icons/faChevronUp';
```

**Comment by jocafi** [comment]
how about the final bundle size when using both variants above (short and long imports) ? Is it the same ?
